iocage_lib/ioc_fetch.py

```diff
@@ -505,10 +505,32 @@ def fetch_http_release(self, eol, _list=False):
             silent=self.silent)
         self.fetch_download(self.files)
         missing = self.__fetch_check__(self.files)
+        missing_attempt = 0
 
-        if missing:
-            self.fetch_download(missing, missing=True)
-            self.__fetch_check__(missing, _missing=True)
+        while True:
+            if not self.files_left:
+                break
+
+            if missing_attempt == 4:
+                iocage_lib.ioc_common.logit(
+                    {
+                        'level': 'EXCEPTION',
+                        'message': 'Max retries exceeded, too many failed'
+                                   ' verifications!'
+                    },
+                    _callback=self.callback,
+                    silent=self.silent)
+
+            _missing = True if missing else False
```

Comment:
That is not necessary, if `missing` is a list simply `if not missing` will satisfy just the same.